configs/15.0/ubuntu/Dockerfile-devel_ppc64le

```diff
@@ -0,0 +1,51 @@
+# Copyright 2021 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG OS_NAME=ubuntu
+ARG OS_RELEASE=focal
+ARG AT_VERSION=15.0
+
+FROM ppc64le/${OS_NAME}:${OS_RELEASE}
+ARG OS_NAME
+ARG OS_RELEASE
+ARG AT_VERSION
+ARG REPO
+
+WORKDIR /root
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN [ -n "$HTTP_PROXY" ] && echo Acquire::http::proxy '"'$HTTP_PROXY'";' >> /etc/apt/apt.conf.d/http_proxy || :
+RUN [ -n "$HTTPS_PROXY" ] && echo Acquire::https::proxy '"'$HTTPS_PROXY'";' >> /etc/apt/apt.conf.d/https_proxy || :
+RUN [ -n "$FTP_PROXY" ] && echo Acquire::ftp::proxy '"'$FTP_PROXY'";' >> /etc/apt/apt.conf.d/ftp_proxy || :
+
+RUN apt update \
+    && apt install -y apt-utils curl gnupg2 \
+        autoconf automake bison byacc ccache cscope ctags flex elfutils libtool m4 make man-db \
```

Comment:
This line is installing a lot of packages.  I think it's safe to remove it and provide a minimal container.

Reply:
Removed.